include/animation/linear_animation_instance.hpp

```diff
@@ -14,7 +15,7 @@ namespace rive
 
 	public:
 		LinearAnimationInstance(LinearAnimation* animation);
-		bool advance(float seconds);
+		bool advance(float seconds, bool didLoop);
```

Comment:
I think this needs to be by reference right? You won't get the result back when calling in. 

```
bool advance(float seconds, bool& didLoop);
```